pruner/find.go

```diff
@@ -18,7 +18,14 @@ func (s *Service) findPruneableHeaders(
 	ctx context.Context,
 	lastPruned *header.ExtendedHeader,
 ) ([]*header.ExtendedHeader, error) {
-	pruneCutoff := time.Now().UTC().Add(-s.window)
+	// TODO(@Wondertan): This should be SyncerHead to ensure the most accurate Head
+	//  but its ok for now and will be fixed in a future release.
+	head, err := s.hstore.Head(ctx)
```

Comment:
There's no danger here really other than estimating a pruneCutoff that's actually **too old** rather than **too new** so I think we're fine here.

Reply:
Eventually w bsync, syncer head will be == store head, no?